typings/index.d.ts

```diff
@@ -3114,6 +3114,29 @@ declare module 'discord.js' {
 
   type Status = number;
 
+  export class Sticker extends Base {
+    constructor(client: Client, data: object);
+    public asset: string;
+    public readonly createdTimestamp: number;
+    public readonly createdAt: Date;
+    public description: string;
+    public format: StickerFormatTypes;
+    public id: Snowflake;
+    public name: string;
+    public packID: Snowflake;
+    public previewAsset: string | null;
+    public tags: string | undefined;
```

Comment:
```suggestion
    public tags: string | null;
```
Should be `null`.

https://github.com/discordjs/discord.js/blob/6315ac77c7fdb6fe8d70e0bca96b41014aca44a7/src/structures/Sticker.js#L56-L60